### ISSUE TITLE
[BUG] copper plate not craftable in the bronze compressor in singleplayer, but it is on the official servers

### ISSUE THREAD

**Issue by FederAndInk** [itsdinkd/AQM2#74]
**Modpack Version**
1.4.1

**Minecraft Version**
1.18.2

**Describe the crash or bug**
The copper plate is not craftable in the bronze compressor in singleplayer, but it is on the official servers.
We can see the recipe in the official servers but not in single player

**Steps to reproduce the behavior:**
try to put a copper ingot into a bronze compressor with steam. or see the recipe in REI

**Expected behavior**
What is the expected outcome?

**Screenshots**

Official servers:

![Screenshot_20220717_204751](https://user-images.githubusercontent.com/24270152/179420443-8e26c619-3bf8-4764-8957-3440cacc7fd5.png)

SinglePlayer:

![Screenshot_20220717_205147](https://user-images.githubusercontent.com/24270152/179420619-77b7f192-1c2e-4b0e-bb17-b4464237815e.png)

**Crash log or latest.log.log**
use pastebin.com for your log

**Additional context**
Any additional context?

**Comment by itsdinkd** [comment]
This is an issue with REI and will need to be reported to their github

**Comment by FederAndInk** [comment]
I mean, its not even craftable in the bronze compressor. It should be right?

**Comment by FederAndInk** [comment]
Just a follow up on this issue, the problem lies in kubejs not loading the scripts in the same order on server vs single player.

The server remove script needs to be loaded before the server unification script, I think they need to be either merged to force the order or the remove script needs to be loaded at the end of the unification script. Or maybe we can specify script dependencies in kubejs?

Basically, the remove script remove the MI copper_plate recipe then the unification script change the recipe from MI copper_plate to TR copper_plate, but it can't be changed if the recipe has been removed first.

**Comment by itsdinkd** [comment]
Great find!!

**Comment by Seceroth** [comment]
You can set the script priority with for example
`//priority: 100`
At the top of a script to set the priority, higher number = higher priority IIRC

**Comment by itsdinkd** [comment]
> You can set the script priority with for example `//priority: 100` At the top of a script to set the priority, higher number = higher priority IIRC

Ahh. Wasn't aware of that, thank you.

**Comment by itsdinkd** [comment]
Fixed in next version!
